runtime/cmd/main.go

```diff
@@ -61,6 +63,7 @@ func main() {
 		ConnectionCacheSize: 100,
 		MetastoreDriver:     conf.DatabaseDriver,
 		MetastoreDSN:        conf.DatabaseURL,
+		QueryCacheSize:      10000,
```

Comment:
These should now be `conf. ConnectionCacheSize` and `conf.QueryCacheSize`

Reply:
changed